entrypoint.py

```diff
@@ -188,8 +188,9 @@
     )
 
     git_repo.git.add('*')
+    deploy_commit_sha = os.environ.get('GITHUB_SHA')
     git_repo.index.commit(
-        '[apt-action] Update apt repo\n\n\napt-action-metadata{}'.format(current_metadata_str)
+        '[apt-action] Update apt repo\n\n\napt-action-metadata{}\n\ndeplying: {}'.format(current_metadata_str, deploy_commit_sha)
```

Comment:
deplying miss-spelled.  Should be 'deploying`. Correct in your fork and I will approve/merge.

Reply:
```suggestion
        '[apt-action] Update apt repo\n\n\napt-action-metadata{}\n\ndeploying: {}'.format(current_metadata_str, deploy_commit_sha)
```